src/click/core.py

```diff
@@ -2454,6 +2456,10 @@ class Option(Parameter):
 
     .. versionchanged:: 8.0.1
         ``type`` is detected from ``flag_value`` if given.
+
+    .. versionchanged:: 8.1.0
```

Comment:
Order change entries for the newest version at the top.

Reply:
reordered